app/src/main/res/values/strings.xml

```diff
@@ -1551,4 +1551,7 @@ Partially means that a wheelchair can enter and use the restroom, but no handrai
     <string name="overlay_sidewalk">Sidewalks</string>
 
     <string name="overlay_street_parking">Street parking</string>
+
+    <string name="quest_accessPointRef_title">What’s the identification number of this?</string>
```

Comment:
I find "here" better than "of this". @peternewman what do you think?

Reply:
"What’s the identification number **here**?", not "of here"

Reply:
Postbox has:
`What’s the identification number of this collection box?`

Which does feel better, as it's not really a place as such, I'm assuming it's lots of different things (well two currently), which would be shown in the description row already.

Plus the old number, when it could be letters too, but we had that for postboxes too.